docs/cawg-identity.md

```diff
@@ -0,0 +1,54 @@
+# Using the CAWG identity assertion
+
+The CAI Rust library includes an implementation of the [1.1 draft of the Creator Assertions Working Group (CAWG) identity assertion specification](https://cawg.io/identity/1.1-draft).
```

Comment:
Another instance of draft that we can probably remove. (Confirm with me tomorrow.)

Reply:
@scouten-adobe Is 1.1 being moved out of "draft" status?  I still see https://cawg.io/identity/1.1-draft/.
Should I hold this PR until this change occurs, or just modify the links/text once it does?